### PR TITLE
[UKRIW-113] run single workflow

### PR DESCRIPTION
**Pull request by mlu-ssoft**
Run single workflow:
- only land cover changes scenario works. Data Set for Land Cover is hardcoded as ATM we don't have Data Sets in Tree View which supports Land Cover changes. This will be changed in another subtask
- validation BE isn't done yet. It will be in another subtask
- types for workflow and presets are adjusted (narrowed)
- presets and functions are using the same endpoint but results are stored under different key and parsed to different interfaces. I though about storing results under the same key, but I think it should be ok if we will call the same enpoint twice. Interfaces for those endpoint differs a lot and I prefer to parse them and throw error if something went wrong over having stored in one key and thinking if changes will affect other type (presets -> functions, functions -> presets)
- there is lot of changed files - I groupped stores under `store` catalog, in most of the files nothing changed there

I didn't cover all cases (eg BE error) to unblock other tasks.

### CODE REVIEW

**Review by KatarzynaPaszko — COMMENTED** [review]
- Data Set I have selected is not reflected in send Payload (I have selected `sentinel-2-l2a` but `stac_collection` send is `"esacci-globallc"`)
Screenshot 2024-10-21 at 09 09 41

**Review by KatarzynaPaszko — COMMENTED** [review]
Call to get functions is triggered too many times and too early:
- it is triggered after switching to AC Mode instead of once user select Functions Node
- It is triggered each time user switch between Workflows tab and Presets tab - it should be fetched once per session, as values there do not change so often

File: libs/shared/design-system/src/lib/input/loading/loading-input.component.tsx
Comment (KatarzynaPaszko) [inline libs/shared/design-system/src/lib/input/loading/loading-input.component.tsx:1]: Where is this component in design ?

File: libs/map/feature-action-creator-panel/src/lib/content/workflow/nodes/function/loading-node.component.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  node: TFunctionNode;
+};
+
+export const LoadingNode = ({ node }: TValueNodeProps) => {
+  return (
+    <Node type={node.type} clickable={false} selected={node.selected}>
+      <LoadingInput />
+    </Node>
+  );
+};
```
[the remark this answers is not in the document]
Comment (mlu-ssoft) [reply libs/map/feature-action-creator-panel/src/lib/content/workflow/nodes/function/loading-node.component.tsx:16]: there is none. I chated with Tobi about it, it is adding spinner into Input. I asked Tobi to update designs

File: libs/map/feature-action-creator-panel/src/lib/content/workflow/workflow-processing-modal.component.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+  const goToHistoryTab = useCallback(() => {
+    reset();
+    setActiveTab('history');
+  }, [reset, setActiveTab]);
+
+  return (
+    <div className='bg-opacity-20 bg-black absolute top-0 left-0 w-full h-full max-h-full max-w-full overflow-hidden flex items-center justify-center'>
+      <div className='mx-8 p-8 bg-bright-light rounded-lg'>
+        <div className='flex flex-row'>
```
[the remark this answers is not in the document]
Comment (mlu-ssoft) [reply libs/map/feature-action-creator-panel/src/lib/content/workflow/workflow-processing-modal.component.tsx:19]: done

File: libs/shared/design-system/src/lib/input/loading/loading-input.component.tsx
Comment (mlu-ssoft) [reply libs/shared/design-system/src/lib/input/loading/loading-input.component.tsx:1]: done

**Review by KatarzynaPaszko — APPROVED** [review]

**Pull request merged**